spec/features/two_factor_authentication/change_factor_spec.rb

```diff
@@ -3,7 +3,7 @@
 feature 'Changing authentication factor' do
   describe 'requires re-authenticating' do
     before do
-      sign_up_and_2fa
+      @user = sign_up_and_2fa
```

Comment:
nit: why use instance variable here instead of a `let(:user)` for consistency with how we do it everywhere else?